Scripts/Services/Assistant/v1/Assistant.cs

```diff
@@ -4444,7 +4444,7 @@ private void OnUpdateDialogNodeResponse(RESTConnector.Request req, RESTConnector
         /// </summary>
         /// <param name="successCallback">The function that is called when the operation is successful.</param>
         /// <param name="failCallback">The function that is called when the operation fails.</param>
-        /// <param name="filter">A cacheable parameter that limits the results to those matching the specified filter. You must specify a filter query that includes a value for `language`, as well as a value for `workspace_id` or `request.context.metadata.deployment`. For more information, see the [documentation](https://console.bluemix.net/docs/services/conversation/filter-reference.html#filter-query-syntax).</param>
+        /// <param name="filter">A cacheable parameter that limits the results to those matching the specified filter. You must specify a filter query that includes a value for `language`, as well as a value for `workspace_id` or `request.context.metadata.deployment`. For more information, see the [documentation](https://cloud.ibm.com/docs/services/conversation/filter-reference.html#filter-query-syntax).</param>
```

Comment:
conversation -> assistant

Reply:
ahh youre right i didnt go far enough in the url. Fixing